docs/MiddlewareConfiguration.md

```diff
@@ -180,6 +180,7 @@ Each element in the ```service_providers``` array must be an object and contain
 * `second_factor_only_nameid_patterns` contains a list of patterns (strings that may contain a '*' wildcard character) that are allowed to use the Second Factor Only mode. E.g. the wilcard pattern `urn:collab:person:example.org:*` matches all NameIDs that start with "urn:collab:person:example.org:". Does nothing if `second_factor_only` is not set to true.
 * ```assertion_encryption_enabled``` must be a boolean value that allows configuring whether or not the assertion that is sent to the SP should be encrypted.
 * ```blacklisted_encryption_algorithms``` contains an array that lists (each as single string-element) algorithms that may not be used for encryption. When left empty, no algorithms are blacklisted. As the gateway currently only allows the " http://www.w3.org/2001/04/xmldsig-more#rsa-sha256" algorithm this option is of little practical use.
+* `use_pdp` optional boolean value, defaults to false, if enabled the PDP endpoint will be called and the required LoA increased if the policy decision obligates a higher LoA
```

Comment:
This sentence does not flow very well. The same goes for the Identity Providers section below.